Sources/Commands/SwiftPackageTool.swift

```diff
@@ -573,6 +573,8 @@ extension SwiftPackageTool {
         }
 
         func run(_ swiftTool: SwiftTool) throws {
+            swiftTool.diagnostics.emit(.warning("Xcode supports Swift Packages directly. 'generate-xcodeproj' is no longer needed and will be deprecated soon."))
```

Comment:
Perhaps the wording "Xcode can open and build Swift Packages directly" would better convey that this doesn't just refer to the ability to add package references to Xcode projects, but also that Xcode can open plain packages.